crates/api_common/src/request.rs

```diff
@@ -233,15 +259,7 @@ async fn generate_pictrs_thumbnail(
   let pictrs_config = context.settings().pictrs_config()?;
 
   if !pictrs_config.cache_external_link_previews {
-    return Ok(
-      proxy_image_link(
-        image_url.clone(),
-        context.settings().pictrs_config()?.image_proxy,
-        context,
-      )
-      .await?
-      .into(),
-    );
+    return Ok(proxy_image_link(image_url.clone(), context).await?.into());
```

Comment:
I commented in the other PR, but this makes it seem like we need to get rid of one of these two pictrs settings bools, since they do the same thing.